pym/_emerge/BinpkgExtractorAsync.py

```diff
@@ -39,6 +46,21 @@ def _start(self):
 			self._async_wait()
 			return
 
+		decompression_binary = varexpand(shlex_split(decomp_cmd)[0], mydict=self.env)
```

Comment:
This `shlex_split(decomp_cmd)[0]` part looks like it might raise IndexError when PORTAGE_BUNZIP2_COMMAND is undefined?